lib/utils/resource-links.js

```diff
@@ -0,0 +1,60 @@
+function isResourceLink (field) {
+  return field && field.sys && field.sys.type === 'ResourceLink'
+}
+
+function getResourceLinkFields (data) {
+  const items = [].concat(...(data.items || []), ...Object.values(data.includes || {}).flat())
+  const fields = items.flatMap(({ fields }) => Object.values(fields).flat())
+  return fields.filter(isResourceLink)
+}
+
+function parseResourceLink (resourceLink) {
+  const urn = resourceLink.sys.urn
+  // For now just assume every resourceLink has type Contentful:Entry
+  const [, spaceId, , entryId] = urn.split('/')
```

Comment:
This was written pretty hastily (by me), is it worth the effort to use a better urn parse function?

Reply:
Since we've already checked that this is a `Contentful:Entry` ResourceLink it should be fine. If we were to change something in the crn pattern it will stop working anyway in the [resolve-response](https://github.com/contentful/contentful-resolve-response/blob/b44ce875c33048d2c19c9a7ca7dd341bd1a237da/index.js#L63).